benchmarks/ampi/alltoall/Makefile

```diff
@@ -37,5 +39,11 @@ alltoall_VPtest_f: alltoall_VPtest.f
 	$(CHARMF) -c alltoall_VPtest.f -swapglobals  $(OPTS)
 	$(CHARMF) -o alltoall_VPtest_f alltoall_VPtest.o -swapglobals $(OPTS) $(LIBS)
 
+test: all
+	$(call run, +p4 ./mpibench +vp2)
```

Comment:
Should this be `+vp4`?

Reply:
Yes, that's right. Fixed in the latest commit. Also added another run with +vp8.